README.rst

```diff
@@ -365,11 +371,101 @@ You will also have to manually remove a few settings::
 
     # MFE ora-grading
     tutor local run lms ./manage.py lms waffle_delete --flags openresponseassessment.enhanced_staff_grader
-
 Finally, restart the platform with::
 
     tutor local launch
 
+
+Template patch catalog
+----------------------
+
+This is the list of all patches used across tutor-mfe (outside of any plugin). Alternatively, you can search for patches in tutor-mfe templates by grepping the source code::
+    
+    git clone https://github.com/overhangio/tutor-mfe
+    cd tutor-mfe
+    git grep "{{ patch" -- tutormfe/templates
+
+mfe-lms-development-settings
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Python-formatted LMS settings in development. Values defined here override the values from `mfe-lms-common-settings <#mfe-lms-common-settings>`_ or `mfe-lms-production-settings <#mfe-lms-production-settings>`_. For an example on the usage of this patch, check out `this section <#mfe-lms-settings>`_.
+
+File changed: ``apps/openedx/settings/lms/development.py``
+
+mfe-lms-production-settings
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Python-formatted LMS settings in production. Values defined here override the values from `mfe-lms-common-settings <#mfe-lms-common-settings>`_. For an example on the usage of this patch, check out `this section <#mfe-lms-settings>`_.
+
+File changed: ``apps/openedx/settings/lms/production.py``
+
+mfe-lms-common-settings
+~~~~~~~~~~~~~~~~~~~~~~~
+Python-formatted LMS settings used both in production and development.
+
+File changed: ``apps/openedx/settings/partials/common_lms.py``
+
+mfe-webpack-dev-config
+~~~~~~~~~~~~~~~~~~~~~~
+Add any additional configurations to be added at the end of the development webpack config file in Javascript format.
```

Comment:
"Add" and "additional" are redundant. I suggest you replace "Add any additional XYZ" by "Add any XYZ" everywhere.